README.md

```diff
@@ -602,7 +602,7 @@ This report is autogenerated and includes tokenizers and detokenizers tests. The
     <tr>
       <td >BPE</td>
       <td >tiiuae/falcon-7b</td>
-      <td >94.51</td>
```

Comment:
looks suspicious..

Reply:
Solved this problem.

I've rerun tests and now for `tiiuae/falcon-7b` pass rate returned to original. The overall pass rate increased.